Packs/OpenSourceVulnerabilities/Integrations/OSV/OSV.py

```diff
@@ -0,0 +1,121 @@
+        headers = self._headers
+        response = self._http_request('post', 'v1new/query', json_data=data, headers=headers)
+
+        return response
+
+
+def osv_get_vuln_by_id_new_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+    id_ = str(args.get('id_', ''))
+
+    response = client.osv_get_vuln_by_id_new_request(id_)
+    command_results = CommandResults(
+        outputs_prefix='OSV.Vulnerability',
+        outputs_key_field='',
+        outputs=response,
+        raw_response=response
+    )
+
+    return command_results
+
+
+def osv_query_affected_by_commit_new_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+    v1query_commit = str(args.get('commit', ''))
+
+    response = client.osv_query_affected_by_commit_new_request(v1query_commit)
+    if response:
+        command_results = CommandResults(
+            outputs_prefix='OSV.VulnerabilityList',
+            outputs_key_field='',
+            outputs=response['vulns'],
+            raw_response=response
+        )
+    else:
+        return_error("Please check if the value provided is correct")
+    return command_results
+
+
+def osv_query_affected_by_package_new_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+    v1query_version = str(args.get('version', ''))
+    v1query_package = str(args.get('packageName', ''))
+    v1query_ecosystem = str(args.get('ecosystem', ''))
+
+    response = client.osv_query_affected_by_package_new_request(v1query_version, v1query_package, v1query_ecosystem)
+    if response:
+        command_results = CommandResults(
+            outputs_prefix='OSV.VulnerabilityList',
+            outputs_key_field='',
+            outputs=response['vulns'],
+            raw_response=response
+        )
+    else:
+        return_error("Please check if the value provided is correct")
+    return command_results
+
+
+def test_module(client: Client) -> None:
+    try:
+        client.osv_get_vuln_by_id_new_request("OSV-2020-111")
+    except Exception as e:
+        raise e
+    return_results('ok')
+
+
+def main() -> None:
+
+    params: Dict[str, Any] = demisto.params()
+    args: Dict[str, Any] = demisto.args()
+    url = params.get('url')
+    verify_certificate: bool = not params.get('insecure', False)
+    proxy = params.get('proxy', False)
+    headers: Dict[str, Any] = {}
+    command = demisto.command()
+    demisto.debug(f'Command being called is {command}')
+
+    try:
+        requests.packages.urllib3.disable_warnings()
+        client: Client = Client(urljoin(url, ''), verify_certificate, proxy, headers=headers, auth=None)
+
+        commands = {
+            'osv-get-vuln-by-id': osv_get_vuln_by_id_new_command,
+            'osv-query-affected-by-commit': osv_query_affected_by_commit_new_command,
+            'osv-query-affected-by-package': osv_query_affected_by_package_new_command
```

Comment:
why do you have the word new in the name of the command function?

Reply:
I used the `demisto-sdk openapi-codegen` The `new` was in the openapi doc. Will remove and update the PR

Reply:
its okay you can keep it as it is, no need to add something